### PR TITLE
feat(section-Image-flush-bottom ): add image-flush-bottom section.

### PR DESCRIPTION
**Pull request by nimithshetty17**
## Issue

Fixes #[MERATIVE-782](https://jira.sdlc.merative.com/browse/MERATIVE-782)

## Description

To create a new section and variations for this called "Image band - flush bottom"

**New**

## Design Specs

> If applicable, add the direct link to the design specs of the component/feature that's part of this PR.

- Figma Link - https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?type=design&node-id=1459-19541&mode=design&t=moJo2KW9FFraRerT-0

## Test URLs
  
- Before (Changes from `main`): https://main--merative2--nimithshetty17.hlx.page/block-library/sections/section-image-band-flush-bottom
- After (Changes from this PR): https://section-image-flush-bottom--merative2--nimithshetty17.hlx.page/block-library/sections/section-image-band-flush-bottom
  
## Testing Instruction

To test the section image flush bottom with brand logo and its responsiveness with respect to the design across all devices.

-

### CODE REVIEW

**Comment by sachinmesh** [comment]
HI,

1. Image outline is missing in all breakpoints except the gradient one.

image

REst is good to go.

@proeung @nimithshetty17 @keith-kaplan @Shalini-SB

**Review by sachinmesh — CHANGES_REQUESTED** [review]
Image outline is missing

**Comment by nimithshetty17** [comment]
@sachinmesh - Please check now.

**Review by sachinmesh — APPROVED** [review]
All issues are fixed, good to go.

@nimithshetty17 @proeung @sharathmrft @Shalini-SB

File: blocks/image-flush-bottom/image-flush-bottom.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
+div:first-child.image-flush-bottom-wrapper,
```
Comment (proeung) [inline blocks/image-flush-bottom/image-flush-bottom.css:1]:
```suggestion
main .section.image-flush-bottom-container > .image-flush-bottom-wrapper
```

Let's keep this defined to the image flush bottom section/classes.

**Comment by nimithshetty17** [comment]
@proeung - have done the changes for the same.

**Review by proeung — APPROVED** [review]
LGTM!

**Pull request merged**